package.json

```diff
@@ -14,6 +14,7 @@
     "dev": "next dev",
     "dev:local": "WEBPACK_WATCH=true next dev",
     "start": "next start",
+    "preview": "next build && cp -r public .next/standalone/ && cp -r .next/static .next/standalone/.next/ && cp -r ca-cert.pem .next/standalone/ && node .next/standalone/server.js",
```

Comment:
This method starts the standalone nexts app. It's needed since `next start` is made for static-generated apps

Reply:
question: this replicates running a production build?

Reply:
yes, the same script as in the `Containerfile` and as explained here: https://nextjs.org/docs/pages/api-reference/next-config-js/output#automatically-copying-traced-files